app/models/master_file.rb

```diff
@@ -170,6 +170,9 @@ def error
   # Generate the waveform after proessing is complete but before master file management
   after_transcoding :generate_waveform
   after_transcoding :update_ingest_batch
+  # Generate and set the poster and thumbnail
+  after_transcoding :set_default_poster_offset
+  after_transcoding :update_stills_from_offset!
```

Comment:
I don't think this is really a big problem, but I think without special handling this will cause batch ingested items to have the poster and thumbnails regenerated since the `after_save` will have already run.